graaljs/graaljs-gradle-webpack-guide/src/main/java/com/example/App.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates.
```

Comment:
Please update the year in all copyright notices:
```suggestion
 * Copyright (c) 2025, Oracle and/or its affiliates.
```